packages/@snowpack/plugin-postcss-transform/plugin.js

```diff
@@ -0,0 +1,21 @@
+const execa = require('execa');
+
+module.exports = function postcssTransformPlugin(_, options) {
+  return {
+    name: '@snowpack/postcss-transform',
+    async transform({fileExt, contents}) {
+      const {input = ['.css'], config, sourceMaps = true} = options;
+      if (!input.includes(fileExt)) return;
+
+      const flags = [];
+      if (config) flags.push(`--config ${config}`);
+
+      const {stdout} = await execa('postcss', flags, {
+        cwd: process.cwd(),
+        input: contents,
+      });
+
+      if (stdout) return stdout;
```

Comment:
Plugin thought: in `transform()`, we only allow strings. However, PostCSS can also generate source maps (`.css.map`). Do we want to allow transform to return an object, too, to generate new files?

Reply:
working through this in #681 !